crates/transaction-pool/src/validate/mod.rs

```diff
@@ -380,11 +376,11 @@ impl<T: PoolTransaction> ValidPoolTransaction<T> {
     }
 }
 
-impl<T: PoolTransaction<Consensus = TransactionSignedEcRecovered>> IntoRecoveredTransaction
+impl<T: PoolTransaction<Consensus: Into<TransactionSignedEcRecovered>>> IntoRecoveredTransaction
```

Comment:
```suggestion
impl<T> IntoRecoveredTransaction where T: PoolTransaction<Consensus: Into<TransactionSignedEcRecovered>>
```
we default to using the where clause syntax when the trait bound is more than one name or there is more than one generic